Automattic/wp-calypso
# Stats: Minor updates to feedback panel

Related to: https://github.com/Automattic/red-team/issues/154

## Proposed Changes

- Adds arrow glyph to review button.
- Adds dismiss button.
- Buttons are full-width on mobile.

#### Full-width buttons

<img width="376" alt="SCR-20240904-pwsi" src="https://github.com/user-attachments/assets/0b83d919-8113-43b2-8d93-f57428e2400d">

#### Panel with dismiss button

<img width="368" alt="SCR-20240904-pxcs" src="https://github.com/user-attachments/assets/b1e29201-1de8-47e3-82bf-90ab30de5043">

## Why are these changes being made?


Part of the User Feedback project.

pejTkB-1Ce-p2

## Testing Instructions



* Launch the Calypso Live site.
* Visit the Traffic page.
* Enable the feature flag: `?flags=stats/user-feedback`
* Confirm the floating panel is visible at the bottom of the window (not the page).
* Confirm the glyph and dismiss button are present.
* Confirm the buttons are full-width on mobile.

## Pre-merge Checklist
